app/assets/javascripts/alert.js

```diff
@@ -0,0 +1,28 @@
+(function () {
+  var float_alert;
+  float_alert = typeof exports !== 'undefined' && exports !== null ? exports : this;
+
+  float_alert.refreshFloatAlertPosition = function refreshFloatAlertPosition() {
```

Comment:
This line can be replaced by:
float_alert.refreshFloatAlertPosition = function () {